### ISSUE TITLE
contribute to book

### ISSUE THREAD

**Issue by sckott** [ropensci/taxize-book#1]
@EDiLD @zachary-foster @gedankenstuecke @arendsee @jarioksa @ibartomeus @jimmyodonnell  @johnbaums  

started this bookdown for taxize pkg. I think we can include `taxizedb` here as well. I think there's sufficient reason to do a bookdown for taxize/taxizedb because of the complexity and importance of the project

if you want to contribute please do. however, there's no pressure at all to contribute. 

I have no intention of publishing this - it seems a nice way to package up documentation

**Comment by EDiLD** [comment]
Great initiative @sckott !

However, I don't know if my free time permits much contributions :(  
In fact, I am happy if I can find the time to take care of the webchem package...

**Comment by gedankenstuecke** [comment]
I’ll be busy traveling to finally defend my PhD this week. But i will have a look afterwards. Also @trvinh might have interest in contributing to this!

**Comment by arendsee** [comment]
Cool project! I'll look it over. Are you interested in collecting usage examples?

**Comment by sckott** [comment]
@EDiLD Okay, thanks for the update

**Comment by sckott** [comment]
@gedankenstuecke ok, thanks for pinging @trvinh

**Comment by sckott** [comment]
@arendsee (there's not much there yet :)) Yes, interested in use cases for sure, probably a chapter on use cases, with as many as we can find

**Comment by johnbaums** [comment]
Sounds great @sckott! I’m a bit time poor these days - 1 year old at home!
Looking forward to taking a look.

**Comment by trvinh** [comment]
@gedankenstuecke , @sckott nice to have this documentation 👍  I will be happy to contribute to this project :-)

**Comment by ibartomeus** [comment]
Same here, very cool idea but not much time. Would be good if you list sections for which you will especially need help (e.g. user cases).

**Comment by sckott** [comment]
thanks @ibartomeus - good idea, haven't sorted out a preliminary set of chapters yet, but will do that when I do

**Comment by arendsee** [comment]
@sckott If you make a `taxizedb` chapter, I'll at very least throw in a few usage examples. I've been intending to write a vignette for the package anyways.

**Comment by sckott** [comment]
@arendsee awesome, thanks. Yes, do plan to have a `taxizedb` chapter

**Comment by zachary-foster** [comment]
Good idea @sckott! I will help in my spare time. Ping me if there is anything specific you want me to do.

**Comment by jimmyodonnell** [comment]
Ditto to the above -- I don't have a ton of spare time, but support the idea and am happy to at least contribute examples for the functions I've worked on.

**Comment by sckott** [comment]
thanks @jimmyodonnell , sounds good.

**Comment by sckott** [comment]
*Everyone* is an author, just FYI, even for contributing only use cases/examples
